### PR TITLE
chore: Update example app

### PR DESCRIPTION
**Pull request by eliaslecomte**

### CODE REVIEW

**Comment by adnan1naeem** [comment]
@eliaslecomte  please merge this pull request if it solved the above issue which you have tagged for android 10 so i can use latest version of this library to verify the issue

**Comment by eliaslecomte** [comment]
> please merge this pull request if it solved the above issue which @eliaslecomte have tagged for android 10 so i can use latest version of this library to verify the issue

You can test react-native-wifi-reborn :-). If you want to test the sample app with android 10, you can just download this source yourself. It doesn't have to be merged. This is the sample app. It doesn't change the react-native-wifi-reborn library.

File: example/RNWifiReborn/App.js [hunk 1]
```diff
@@ -23,33 +24,68 @@ const App = () => {
...
+    }
+  };
+
+  const connectWithWifi = async () => {
     try {
       const data = await WifiManager.connectToProtectedSSID(
         ssid,
         password,
-        isWep,
+        false,
```
Comment (adnan1naeem) [inline example/RNWifiReborn/App.js:67]: I have applied this change but still not beneficial.It gets connected but throws error in callback

Comment (eliaslecomte) [reply example/RNWifiReborn/App.js:67]: Did you clean via gradle? (cd android && ./gradlew clean). What exception do you get?

Comment (adnan1naeem) [reply example/RNWifiReborn/App.js:67]: https://github.com/JuanSeBestia/react-native-wifi-reborn/issues/30

Comment (eliaslecomte) [reply example/RNWifiReborn/App.js:67]: In that case I assume you are using Android 7/8/9? What if you remove the wifi network first from Settings?

**Comment by adnan1naeem** [comment]
Yes it is working fine on android 7 till 9 but failing at 10.how to target
for  android 7 to 10

**Comment by eliaslecomte** [comment]
> Yes it is working fine on android 7 till 9 but failing at 10.how to target for android 7 to 10

In your android/app/build.gradle you need to have
`android {
    compileSdkVersion 29
    defaultConfig {
        targetSdkVersion 29`

**Pull request merged**